R/cache.R

```diff
@@ -50,17 +50,12 @@ font_cache_housing <- function() {
   )
 }
 
-# Intentionally mimics `tools::R_user_dir("thematic", "cache")` (coming in R 4.0)
-# Copyright (C) 2020 The R Core Team
-# https://github.com/wch/r-source/blob/trunk/src/library/tools/R/userdir.R
+
 thematic_cache_dir <- function() {
-  home <- normalizePath("~")
-  path <- if (.Platform$OS.type == "windows") {
-    file.path(Sys.getenv("LOCALAPPDATA"), "R", "cache")
-  } else if (Sys.info()["sysname"] == "Darwin") {
-    file.path(home, "Library", "Caches", "org.R-project.R")
+  user_dir <- if (getRversion() >= "4.0") {
+    tools::R_user_dir
   } else {
-    file.path(home, ".cache")
```

Comment:
Can't you just use `backports::R_user_dir`?

Reply:
Nope, it only exports `backports::import` which is intended to be used in `.onLoad()`